packages/nextjs-template/context/DendronProvider.tsx

```diff
@@ -0,0 +1,47 @@
+import { convertLegacyProps } from "antd/lib/button/button";
```

Comment:
remove?

Reply:
Removed with a force push amend to the mermaid commit.